Dockerfile

```
@@ -6,12 +6,12 @@ RUN apt-get install -y curl
 RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
 ENV PATH="${PATH}:/root/.poetry/bin"
 COPY ./poetry.toml ./pyproject.toml poetry.lock ./ 
-RUN poetry install
+RUN poetry config virtualenvs.create false --local && poetry install
 
 
 FROM base AS production
 COPY ./todo_app ./todo_app/
-CMD [ "poetry", "run", "gunicorn", "--bind", "0.0.0.0:5000", "todo_app.wsgi:app" ]
+CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```

This works and is necessary for Heroku, but note that it breaks your Readme instructions since you aren't telling users to set a PORT environment variable! Setting a default value so that we don't need to adjust those instructions would be nice, one way to do so is through Docker:
```suggestion
ENV PORT=5000
CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```
This way it will stay as 5000 *unless* someone passes an alternative environment variable in which case it will respect that